composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "roave/better-reflection",
+    "name": "gvanbeck/better-reflection",
```

Comment:
Why are you renaming this...?

Reply:
Please I was not meaning to do pullrequests. Please decline them all for today

Sorry to bother